Site/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "apidb",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Just to clarify: the `*Website` and `install` repos aren't published to npm, so their versions don't have to be incremented